.pylintrc

```diff
@@ -152,7 +152,9 @@ disable=abstract-method,
         raise-missing-from,
         super-with-arguments,
         unnecessary-pass,
-        useless-object-inheritance
+        useless-object-inheritance,
+        consider-using-f-string,
+        consider-using-in
```

Comment:
These were newly discovered after I removed the Python compatibility support. Since there are many occurrences I decided to ignore it, TODO is already ready above.